src/mongocxx/exception/error_code.hpp

```diff
@@ -25,6 +25,7 @@ enum class error_code : std::int32_t {
 	k_invalid_client_object = 1,
 	k_invalid_collection_object,
 	k_invalid_database_object,
+        k_invalid_parameter,
```

Comment:
The weird spacing is an artifact - apparently this list is using leading tabs? I've cleaned it up, but we can leave the real fix for clang-format after rc0